xbmc/pvr/PVRDatabase.cpp

```diff
@@ -45,7 +45,7 @@ void CPVRDatabase::CreateTables()
 {
   CLog::Log(LOGINFO, "PVR - %s - creating tables", __FUNCTION__);
 
-  CLog::Log(LOGDEBUG, "PVR - %s - creating table 'channels'", __FUNCTION__);
+  CLog::Log(LOGDEBUG, "CPVRDatabase - %s - creating table 'channels'", __FUNCTION__);
```

Comment:
you grepped for the string?

I've changed them to "PVR" in the past because the lines are unique already, we don't do that elsewhere in the code (except perhaps for some rare places) and I was thinking about removing the method names from most log lines too earlier. We generate way too much useless spam when creating logs.